docs/getting-started.md

```diff
@@ -1,21 +1,162 @@
+
+# Keyboard Shortcuts
+
+Sometimes it can to take forever to manually move our cursor where we want it - that's why Zulip Terminal has keyboard shortcuts! Type <kbd>?</kbd> to open **help** and view the keyboard shortcuts that are available to you at your cursor's current position. **Help** (<kbd>?</kbd>) updates automatically as you move your cursor around (some shortcuts aren't available in certain sections). You can also view a list of [keyboard shortcuts here.](https://github.com/zulip/zulip-terminal#hot-keys)
+
+![Screenshot Showing Help](getting-started-imgs/help.png)
+
+Let's try out a few of these shortcuts out!
+
++ First, type <kbd>P</kbd> to view your Private Messages (PMs).
+
++ Second, type <kbd>#</kbd> to view all messages you were mentioned (@'ed) in.
+
++ Third, type <kbd>/</kbd> to jump to the search bar at the top of the in the middle column. Type <kbd>esc</kbd> to move your cursor out of the search box so we can use our keyboard shortcuts again (otherwise you'd just be typing them in the search box!).
+
+
+# Narrowing
+
+You'll be hearing the term 'narrow' a lot in in Zulip. What does it mean?
+
++ **Noun**: A narrow is a set of filters for Zulip messages, that can be based on many different factors (like sender, stream, topic, search keywords, etc.).
++ **Verb**: The process of navigating to a more specific narrow. For example, to go from viewing a stream to a topic within that stream.
+
+## Your Current Narrow
+
+Your current narrow is displayed at the top of Zulip Terminal in the middle column, see the image below for details. If you ever get lost, just look here.
+![Screenshot Showing Current Narrow](getting-started-imgs/current-narrow.png)
+
+## Narrow to a Stream
+To 'narrow to a stream' means to set the middle column to only show messages present in that stream. Let's try narrowing to the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream. [Move your Cursor](#Your-Cursor) to the left column of Zulip Terminal and then move your cursor down (<kbd>Down</kbd>/<kbd>j</kbd>) until it rests on "# test here". Press <kbd>Enter</kbd> to narrow to the stream.
+
+Check out the top of the middle column, your current narrow should now be "test here" like in the image below.
+
+![Screenshot Showing Narrow to # test here stream](getting-started-imgs/narrow-stream.png)
+
+## Narrow to a Topic
+
+First, let's move our cursor back to the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream in the left column - otherwise the following keyboard shortcut won't work. With our cursor on the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream, type <kbd>t</kbd> to toggle on a list of topics within that stream. Move your cursor to a random topic and then press <kbd>Enter</kbd> to narrow to that topic. You've successfully narrowed to a topic! Notice how at the top of the middle column your current narrow is dispalyed as "test here: topic you chose". 
```

Comment:
A small typo in the last sentence-  
`your current narrow is dispalyed` --> `your current narrow is displayed`